vars/createArgusTestRun.groovy

```diff
@@ -2,20 +2,22 @@
 
 def call(Map params) {
     def test_config = groovy.json.JsonOutput.toJson(params.test_config)
-    sh """#!/bin/bash
-        set -xe
+    retry(3) {
```

Comment:
Where does this func come from?

Reply:
from jenkins
https://www.jenkins.io/doc/pipeline/steps/workflow-basic-steps/#retry-retry-the-body-up-to-n-times